src/bin/pgcopydb/catalog.c

```diff
@@ -7700,6 +7704,169 @@ catalog_count_summary_done_fetch(SQLiteQuery *query)
 }
 
 
+/*
+ * catalog_add_timeline_history inserts a timeline history entry to our
+ * internal catalogs database.
+ */
+bool
+catalog_add_timeline_history(void *ctx, TimelineHistoryEntry *entry)
+{
+	TimelineHistoryContext *context = (TimelineHistoryContext *) ctx;
+	DatabaseCatalog *catalog = context->source;
```

Comment:
Can we pass just the `DatabaseCatalog *catalog` as the first argument to this function?